try{
    throw new Error("Something is wrong");
}
catch(error){
    console.log(error);
}